docs/django-widget.rst

```diff
@@ -10,23 +10,31 @@ Settings
 --------
 
 Besides required ``pub_key``, ``secret`` settings there are optional settings,
-for example, ``widget_version`` or ``widget_build``:
+for example, ``widget_version`` or ``widget_variant``:
 
 .. code-block:: python
 
     UPLOADCARE = {
         'pub_key': 'demopublickey',
         'secret': 'demoprivatekey',
-        'widget_version': '3.x',  // ~= 3.0 (latest)
-        'widget_build': 'min',  // without jQuery
+        'widget_version': '0.26.0',
```

Comment:
Is it possible to make the version dynamic?

Or, @nd0ut is there some "latest" version option?

Reply:
Yes, but what if there are breaking changes in future releases?

While I'm in favor of using the `latest` version, there are potential downsides. Should we consider using `0.x` instead?

Reply:
Changed it from `0.x` to `0.30.2`